doc/articles/splash-screen.md

```diff
@@ -1,97 +1,68 @@
 ---
-uid: Uno.Development.Splashscreen
+uid: Uno.Development.SplashScreen
 ---
 
-# How to add a Splash Screen
+# How to manually add a splash screen
 
-This article covers how to add a splash screen to your application if you don't want to use Uno.Resizetizer to handle your Splash Screen.
+Projects created using Uno Platform 4.8 or later have the Uno.Resizetizer [package](https://www.nuget.org/packages/Uno.Resizetizer) installed by default. Simply provide an SVG file, and the tool handles the task of generating various image sizes. That package updates the build process to automate configuring a splash screen for each platform. 
```

Comment:
Should we link to this part of the docs? https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?tabs=classlib%2CAndroid#unosplashscreen

Reply:
Nevermind I just saw you did that a bit later on :)